mdoc-js/src/main/scala/mdoc/modifiers/ScalajsClassloader.scala

```diff
@@ -0,0 +1,29 @@
+package mdoc.modifiers
+
+import java.net.URL
+import java.net.URLClassLoader
+
+final class FilteringClassLoader(parent: ClassLoader) extends ClassLoader(parent) {
+  private val parentPrefixes = List(
```

Comment:
nit: the `loadClass` is a fairly hot path so I'd try to avoid `List` here
```suggestion
  private val parentPrefixes = Array(
```